wicket-core/src/main/java/org/apache/wicket/core/request/handler/PageProvider.java

```diff
@@ -54,15 +52,14 @@
 
 	private final Integer pageId;
 
-	private IPageSource pageSource;
-
-	private transient IRequestablePage pageInstance;
-	private boolean pageInstanceIsFresh;
+	private transient IPageSource pageSource;
```

Comment:
IMO using `transient` is just delaying the problem.
`PageProvider` should not be serializable (IClusterable).
If it is serialized and then deserialized then most probably some call with fail with NullPointerException because of those transient fields.